Documentation/UploadBinaryData.md

```diff
@@ -48,58 +83,76 @@ need to be followed:
      * Spacing should be different from ones, and it should be anisotropic.
      * Direction should be different from identity.
 
-Using Girder to get the SHA512 hash
------------------------------------
+Upload new testing data
+-----------------------
 
 ### Prerequisites
 
-The [data.kitware.com] server is an ITK community resource where any community
-member can upload binary data files. There are three methods available to
-upload data files:
+[web3.storage] is a decentralized IPFS storage
+provider where any ITK community member can upload binary data files.
+There are two methods available to upload data files:
+
+1.  The CMake ExternalData Web3 upload browser interface.
+2.  The <span class="title-ref">w3</span> command line executable that
+    comes with the [@web3-storage/w3] Node.js NPM package.
+
+Once files have been uploaded to your account, they will be publicly
+available and accessible since data is content addressed on the IPFS
+peer-to-peer network. At release time, the release manager will upload
+and archive repository data references in other redundant storage
+locations.
+
+### Upload Via the Web Interface
+
+Use the [CMake ExternalData Web3
+Upload](https://cmake-w3-externaldata-upload.on.fleek.co/) tool to
+upload your data to the InterPlanetary Filesystem and download the
+corresponding CMake content link file.
+
+![[CMake ExternalData Web3
+Upload](https://cmake-w3-externaldata-upload.on.fleek.co/)](CMakeW3ExternalDataUpload.png)
 
-  1. The [UploadBinaryData.sh] shell script.
-  1. The [Girder web interface].
-  2. The `girder-cli` command line executable that comes with the
-     [girder-client] Python package.
+Add the file to the examples repository in your example's directory.
+Next time CMake configuration runs, it will find the new content link.
+During the next project build, the data file corresponding to the
+content link will be downloaded into the build tree.
 
-Before uploading data, please visit [data.kitware.com] and register for an
-account.
+### Upload Via CMake and Node.js CLI
 
-Once files have been uploaded to your account, they will be publicly available
-and accessible since data is content addressed. Specifically, the
-[hashsum_download] plugin in Girder looks through all public (or private if
-authenticated) data for files with the given hash. Thus, so as long as the file
-is publicly available somewhere on [data.kitware.com], ITK will be able to
-retrieve the corresponding file.
+Install the <span class="title-ref">w3</span> CLI with the
+[@web3-storage/w3] [Node.js] package:
 
-At release time, the release manager will upload and archive repository data
-references in the [ITK collection] and other redundant storage locations.
+``` shell
+$ npm install --location=global @web3-storage/w3
+```
 
+Login in and create an API token at
+[web3.storage] then pass it into \`w3 token\`:
```

Comment:
Not sure the syntax of \`w3 token\` is accurate for highlighting if that was the intent.

Reply:
Good eye, fixed.